Lib/ast.py

```diff
@@ -578,7 +583,19 @@ def _dims_getter(self):
 def _dims_setter(self, value):
     self.elts = value
 
-Tuple.dims = property(_dims_getter, _dims_setter)
+Tuple.dims = property(_dims_getter, _dims_setter, doc='Deprecated. Use elts instead.')
+
+class Suite(mod):
+    """Unused AST node class."""
+
+class AugLoad(expr_context):
+    """Unused AST node class."""
+
+class AugStore(expr_context):
+    """Unused AST node class."""
+
+class Param(expr_context):
+    """Unused AST node class"""
```

Comment:
Should we deprecate this as well?

Reply:
I used word "deprecated" if it was used in past Python releases and the user code can use them for compatibility. I used word "unused" if it was not used in Python (at least in Python 3) and there is no reason to use it in the user code.

Reply:
But aren't them will be removed in time or stay them forever? If they will, wouldn't it be good to start doing warnings?

Reply:
It would be better to start emitting warnings when the user code can just drop these classes. Currently Param is needed for 2.7, and some projects still support it.

Reply:
> Currently Param is needed for 2.7, and some projects still support it.

Oh, that makes sense.